sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -540,8 +540,9 @@ def upload_blob(
             value specified in this header, the request will fail with
             MaxBlobSizeConditionNotMet error (HTTP status code 412 - Precondition Failed).
         :keyword int max_concurrency:
-            Maximum number of parallel connections to use when the blob size exceeds
-            64MB.
+            Maximum number of parallel connections to use when the blob size exceeds 64MB.
```

Comment:
Sorry but I see this as an opportunity to fix this as well: Can change the wording a bit to "Maximum number of parallel connections to use when transferring the blob in chunks."

We want to remove the 64 MiB from here because that is a configurable setting, and it may not actually be the default for all SDKs.

Reply:
This is definitely a great call out, will implement the change. Thanks :D